soccer_object_detection/src/object_detect_node.py

```diff
@@ -88,7 +88,8 @@ def callback(self, msg: Image):
             bbs_msg = BoundingBoxes()
             for prediction in results.xyxy[0]:
                 x1, y1, x2, y2, confidence, img_class = prediction.cpu().numpy()
-                if img_class == SOCCER_BALL:
+                print(x1, y1, x2, y2, confidence, img_class)
```

Comment:
remove print statement it is spamming the logs
use rospy.loginfo_throttle(seconds, blablala)